Client/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -386,7 +386,7 @@ class TopToolbarView: UIView, ToolbarProtocol {
         if search {
             locationTextField?.text = text
             // Not notifying when empty agrees with AutocompleteTextField.textDidChange.
-            delegate?.topToolbar(self, didEnterText: text)
+            delegate?.topToolbar(self, didSubmitText: text)
```

Comment:
What is this change needed for? 
Currently it seems to break other part of the app when you are on google.com for example, and tap on url bar, it reloads the app instead of focusing on url bar

Reply:
I noticed that without this change, view is stuck with empty url bar and doesn't submit searching. Thanks for your comment, I haven't realized it breaks other part of the app, will try to solve that in different way.